definitions/divorce/json/CaseEventToComplexTypes/CaseEventToComplexTypes-nonprod.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "LiveFrom": "20/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "ID": "OrganisationPolicy",
+    "CaseEventID": "tempSuperUserTestEvent",
+    "CaseFieldID": "RespondentOrganisationPolicy",
+    "ListElementCode": "Organisation.OrganisationID",
+    "EventElementLabel": "",
```

Comment:
Shall we remove those unused fields?

Reply:
Possibly, that was just a copy and paste I am not sure whether there was any reason it was left like this.